model.py

```diff
@@ -879,38 +883,44 @@ def plot_gmm_prior(stellar_type_prior):
     lnp = stellar_type_prior.ln_prob(param_grid).numpy()
     lnp.shape = s
 
-    fig,ax_arr = plt.subplots(1,3, figsize=(6,2))
+    for suffix,norm in [('linear',Normalize),('log',LogNorm)]:
+        fig,ax_arr = plt.subplots(1,3, figsize=(6,2), layout='constrained')
```

Comment:
`layout='constrained'` handles simple plots like this well, without the need to fiddle with `subplots_adjust`. The only downside is that it requires a relatively recent version of `matplotlib`.